src/Tribe/Events/Views/V2/Hooks.php

```diff
@@ -43,20 +44,52 @@ public function register() {
 	 *
 	 * @since 4.10.9
 	 *
-	 * @param array $folders The current list of folders that will be searched template files.
+	 * @param array           $folders  The current list of folders that will be searched template files.
+	 * @param Tribe__Template $template Which template instance we are dealing with.
 	 *
 	 * @return array The filtered list of folders that will be searched for the templates.
 	 */
-	public function filter_template_path_list( array $folders = [] ) {
-		$folders[] = [
-			'id'       => 'event-tickets',
-			'priority' => 17,
-			'path'     => Plugin::instance()->plugin_path . 'src/views/v2',
+	public function filter_template_path_list( array $folders, Tribe__Template $template ) {
+		/** @var Plugin $main */
+		$main = tribe( 'tickets.main' );
+
+		$path = (array) rtrim( $main->plugin_path, '/' );
+
+		// Pick up if the folder needs to be added to the public template path.
+		$folder = $template->get_template_folder();
+
+		if ( ! empty( $folder ) ) {
+			$path = array_merge( $path, $folder );
+		}
+
+		$folders['event-tickets'] = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: e838b5651f9d0d8de8ffa20a6420be8d</sub>